### DISCUSSION TITLE
Make UI sliders TRULY dragable

### DISCUSSION THREAD

**Discussion by liesened** [ppy/osu#18938] [Ideas] [answered]
Currently, the center of a slider box tied to the cursor position which makes it hard to adjust the value when clicking on either side of the slider, as shown below.

https://user-images.githubusercontent.com/101940285/176500450-c826075f-969d-4b72-bb17-f52796964314.mp4

This is especially annoying when trying to adjust the slider with a touchscreen or tablet.

So, I suggest that the slider position should be updated according to cursor movement length rather its position.

**Comment by peppy** [comment, marked as answer]
Addressed via https://github.com/ppy/osu/pull/18940
